mapper/orm/src/main/java/org/hibernate/search/mapper/orm/massindexing/spi/MassIndexer.java

```diff
@@ -4,7 +4,7 @@
  * License: GNU Lesser General Public License (LGPL), version 2.1 or later
  * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
  */
-package org.hibernate.search.mapper.orm;
+package org.hibernate.search.mapper.orm.massindexing.spi;
```

Comment:
Why SPI? Users have to use the mass indexer, this particular interface should be API.

The build should have failed because this class is referenced from API. I will investigate. In the meantime, please move this interface to API.